src/optimizer/optimizer.h

```diff
@@ -84,6 +84,8 @@ class regu_variable_node;
 #define PLAN_DUMP_ENABLED(level)	((level) >= 0x100)
 #define SIMPLE_DUMP(level)		((level) & 0x100)
 #define DETAILED_DUMP(level)		((level) & 0x200)
+#define CHECK_VALID_EXECUTION(level)	((level & 0x00ff) > 0 && (level & 0x00ff) < 3)
+#define CHECK_VALID_PLAN(level)		(((level>>8) & 0x00ff) >= 0 && ((level>>8) & 0x00ff) < 3)
```

Comment:
```suggestion
#define CHECK_INVALID_PLAN(level)		(((level) & 0xff) < 3)
```
이 두 개의 매크로를 사용하는 루틴은 모두 "!(CHECK_VALID_EXECUTION (level) && CHECK_VALID_PLAN (level))" 형태입니다. 굳이 두 개의 매크로를 사용할 필요 없이, 하나의 매크로로 INVALID를 체크하는 것이 좋겠습니다.

Reply:
하위 1 byte(execution 여부)와 상위 1 byte (plan off/simple/detail)가 분리되어 있어, 명확한 의미를 주기 위해 매크로를 2개로 나누었습니다.
제 생각에는 두개의 매크로를 하나로 합치는 것보다는 다음과 같은 매크로를 추가하는 것은 어떨까요 ?

#define CHECK_INVALID_OPTIMIZATION_LEVEL(level) (!(CHECK_VALID_EXECUTION (level) && CHECK_VALID_PLAN (level)))

Reply:
네, 좋습니다.